comparisons/navigation/accordion/html.html

```diff
@@ -1,19 +1,19 @@
-<div class="togglebox">
+<div class="togglebox" role="tablist">
```

Comment:
I tested this locally and it doesn't seem to make the toggle keyboard accessible still :/